### ISSUE TITLE
Highlight in docs that schema checks is not enabled by default

### ISSUE THREAD

**Issue by pwalsh** [frictionlessdata/goodtables-py#185]
```
@goodtables.check('expected-range', type='schema', context='body', after='non-castable-value')
def expected_range(errors, columns, row_number,  state=None):
    for column in columns:
        errors.append({
            'code': 'expected-range',
            'message': 'Expected range',
            'row-number': row_number,
            'column-number': column['number'],
        })
```

```
inspector = goodtables.Inspector(custom_checks=[expected_range])
inspector.inspect('data/israel-muni/tel-aviv-2013.csv')
```

Output:

```
{'error-count': 0,
 'errors': [],
 'table-count': 1,
 'tables': [{'error-count': 0,
   'errors': [],
   'headers': ['PARENT',
    'PARENT SCOPE',
    'CODE',
    'NAME',
    'NAME_EN',
    'NAME_RU',
    'NAME_AR',
    'BUDGET',
    'ACTUAL'],
   'row-count': 1000,
   'source': 'data/israel-muni/tel-aviv-2013.csv',
   'time': 0.03,
   'valid': True}],
 'time': 0.031,
 'valid': True}
```

Expected:

Custom check error on each column (cell?)

**Comment by roll** [comment]
@pwalsh 
You've marked it as `type=schema` but following our decision schema checks will be processed only if:
- schema is passed
- infer_schema is True

**Comment by pwalsh** [comment]
@roll decisions are made and forgotten all the time :). Can you write this in the documentation somewhere as well please (if it is there, sorry, I missed it).

**Comment by roll** [comment]
MERGED (action - add a note to readme) #105
